decidim-core/app/models/decidim/push_notification_message.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Decidim
+  # A messsage from a conversation that will be sent as a push notification
+  class PushNotificationMessage
+    class InvalidActionError < StandardError; end
+
+    def initialize(recipient:, conversation:, message:)
+      @recipient = recipient
+      @conversation = conversation
+      @message = message
+    end
+
+    attr_reader :recipient, :conversation, :message
+
+    include SanitizeHelper
```

Comment:
For consistency with other classes, the modules inclusions should be on top. 
```suggestion
  class PushNotificationMessage
    class InvalidActionError < StandardError; end

    include SanitizeHelper

    def initialize(recipient:, conversation:, message:)
      @recipient = recipient
      @conversation = conversation
      @message = message
    end

    attr_reader :recipient, :conversation, :message

```